cpp/src/utilities/cython.cpp

```diff
@@ -27,28 +28,161 @@ namespace cython {
 // Populates a graph_container_t with a pointer to a new graph object and sets
 // the meta-data accordingly.  The graph container owns the pointer and it is
 // assumed it will delete it on destruction.
-//
-// FIXME: Should local_* values be void* as well?
 void populate_graph_container(graph_container_t& graph_container,
-                              legacyGraphTypeEnum legacyType,
-                              raft::handle_t const& handle,
-                              void* offsets,
-                              void* indices,
+                              raft::handle_t& handle,
+                              void* src_vertices,
+                              void* dst_vertices,
                               void* weights,
-                              numberTypeEnum offsetType,
-                              numberTypeEnum indexType,
+                              void* vertex_partition_offsets,
+                              numberTypeEnum vertexType,
+                              numberTypeEnum edgeType,
                               numberTypeEnum weightType,
-                              int num_vertices,
-                              int num_edges,
-                              int* local_vertices,
-                              int* local_edges,
-                              int* local_offsets,
+                              int num_partition_edges,
+                              size_t num_global_vertices,
+                              size_t num_global_edges,
+                              size_t row_comm_size,  // pcols
+                              size_t col_comm_size,  // prows
                               bool transposed,
                               bool multi_gpu)
 {
   CUGRAPH_EXPECTS(graph_container.graph_ptr_type == graphTypeEnum::null,
                   "populate_graph_container() can only be called on an empty container.");
 
+  bool do_expensive_check{true};
```

Comment:
Should this be `False` by default?

Reply:
yes, thank you, that was left on for debugging. Fixed in latest commit.